src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/tests/test_custom.py

```diff
@@ -13,16 +13,19 @@
 
 from msrestazure.azure_exceptions import CloudError
 
-from azure.cli.command_modules.acs.custom import merge_kubernetes_configurations, _acs_browse_internal, _add_role_assignment
-from azure.mgmt.compute.models import ContainerServiceOchestratorTypes, ContainerService, ContainerServiceOrchestratorProfile
+from azure.cli.command_modules.acs.custom import merge_kubernetes_configurations, \
+    _acs_browse_internal, _add_role_assignment
```

Comment:
Question:
How about the form `(Dep1, Dep2, Dep3)` then you don't need the backslash `\`.
I prefer the `(...)` form. Your thoughts?